src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -298,41 +298,6 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
         );
     }
 
-    /// @inheritdoc IFM_BC_Bancor_Redeeming_VirtualSupply_v1
-    function calculatePurchaseReturn(uint _depositAmount)
```

Comment:
Can you remind me of the reason to move these two to the upstream contract? I'm not against it, just curious

Reply:
The reason being is that we need this function in each implementation, and as far as I've seen with implementing both formula contracts, we use the internal issuance/redeem supply wrapper for calculating the return value. Next to this, we also need to deduct the fees from the return values. So to me it made sense to standardize this in the upstream contract, and leave the option for the implementation to change it if needed